tofd/pars/R3BTofDCal2HitPar.cxx

```diff
@@ -297,1099 +352,2668 @@ void R3BTofDCal2HitPar::Exec(Option_t* option)
+			    // fhTot1vsTot2[iPlane - 1][iBar - 1]->Fill(top_tot, bot_tot);
+			    fhLogTot1vsLogTot2[iPlane - 1][iBar - 1]->Fill(TMath::Log(top_tot), TMath::Log(bot_tot));
+			    fh_tofd_TotPm[iPlane - 1][0]->Fill(iBar, top_tot);
+			    fh_tofd_TotPm[iPlane - 1][0]->Fill(-iBar - 1, bot_tot);
+			    fh_tofd_TotPm[iPlane - 1][1]->Fill(iBar, top_tot  * par->GetToTOffset2());
+			    fh_tofd_TotPm[iPlane - 1][1]->Fill(-iBar - 1, bot_tot * par->GetToTOffset1());
+			    fh_tofd_TotPm[iPlane - 1][2]->Fill(iBar, top_tot  / par->GetToTOffset2());
+			    fh_tofd_TotPm[iPlane - 1][2]->Fill(-iBar - 1, bot_tot / par->GetToTOffset1());
+				//cout<<"ToT_t "<<top_tot<<" par_a "<<par->GetToTOffset3()<<" par_b "<<par->GetToTOffset4()<<" | ToT_b "<<bot_tot<<" par_a "<<par->GetToTOffset1()<<" par_b "<<par->GetToTOffset2()<<endl;
+			
+			    // Time differences of one paddle; offset  histo
+
+			    if((sqrt(Ctop_tot*Cbot_tot)>fToTMin && sqrt(Ctop_tot*Cbot_tot)<fToTMax) || ToTexeption){
+				    fhTdiff[iPlane - 1]->Fill(iBar, tdiff);
+			    }
+			    // offset histo via ToT
+			    /*if (fTofdY == 0.)
+			      {
+			      R3BLOG(debug, "Will prepare for offset and sync calculation");
+			      posToT = TMath::Log(top_tot / bot_tot);
+			      }
+			      else
+			      {*/
+			    auto posToT = TMath::Log((Ctop_tot) / (Cbot_tot));
+
+			    //}
+		
+				cout<<sqrt(Ctop_tot*Cbot_tot)<<" "<<fToTMin<<" "<<fToTMax<<" "<<Ctop_tot<<" "<<Cbot_tot<<" "<<ToTexeption<<endl;
+			    if((sqrt(Ctop_tot*Cbot_tot)>fToTMin && sqrt(Ctop_tot*Cbot_tot)<fToTMax) || ToTexeption){
+				    fhSqrtQvsPosToT[iPlane - 1][iBar - 1]->Fill(posToT, sqrt(Ctop_tot * Cbot_tot));
+				    //fhTot1vsPos[iPlane - 1][iBar - 1]->Fill(posToT, Cbot_tot);
+				    //fhTot2vsPos[iPlane - 1][iBar - 1]->Fill(posToT, Ctop_tot);
+			    }
+			    // Time of hit
+			    if((sqrt(Ctop_tot*Cbot_tot)>fToTMin && sqrt(Ctop_tot*Cbot_tot)<fToTMax) || ToTexeption){
+				    auto THit = (top_ns + bot_ns) / 2. - par->GetSync();
+				    fhTsync[iPlane - 1]->Fill(iBar, THit);
+			    }
+			    // Tof with respect LOS detector
+			    if((sqrt(Ctop_tot*Cbot_tot)>fToTMin && sqrt(Ctop_tot*Cbot_tot)<fToTMax) /*|| ToTexeption*/){
+				    auto tof = fTimeStitch->GetTime((top_ns + bot_ns) / 2. - fHeader->GetTStart(), "tamex", "vftx");
+				    if(fParameter == 2){
+						fhTofsync[iPlane-1]->Fill(iBar,tof);
+				    	fh1_tofsync[iPlane - 1][iBar - 1]->Fill(tof);	
+					}
+					else{
+						fhTofsync[iPlane-1]->Fill(iBar,tof - par->GetTofSyncOffset());
+				    	fh1_tofsync[iPlane - 1][iBar - 1]->Fill(tof - par->GetTofSyncOffset());
+					}
+			    }
+			}
+		    if(fParameter==0){
+
+			    if((sqrt(Ctop_tot*Cbot_tot)>fToTMin && sqrt(Ctop_tot*Cbot_tot)<fToTMax) || ToTexeption){
+				    fh1_walk_t[iPlane-1][iBar-1]->Fill(Ctop_tot,top_ns);
+				    fh1_walk_b[iPlane-1][iBar-1]->Fill(Cbot_tot,bot_ns);
+			    }
+		    }
+
+
+		    else if ((fTofdQ > 0 && fParameter > 1 && fParameter != 10) || fParameter == 3)
+		    {
```

Comment:
Is this condition every entered?

The initial if statement is (fTofd>0 && fParameter>1 && fParameter!=10). fParameter==3 is greater than 1 and not 10